src/test/resources/org/prebid/server/it/storedrequests/test-amp-stored-request.json

```diff
@@ -6,20 +6,27 @@
       "banner": {
         "format": [
           {
-            "w": 300,
-            "h": 600
+            "w": 980,
+            "h": 120
           }
         ],
         "w": 300,
         "h": 600
       },
+      "tagid": "overwrite-tagId",
+      "secure": 1,
       "ext": {
-        "rubicon": {
+        "data": {
+          "gam-key1": "val1",
+          "gam-key2": "val2"
+        },
+        "tid": "6b8762d8-26ec-45bc-821b-8d6a72fdcd55",
+        "generic": {
           "accountId": 5001,
           "siteId": 6001,
           "zoneId": 7001
         },
-        "appnexus": {
+        "genericAlias": {
```

Comment:
We don't need appnexus properties 
Just some `"property1":"value1"`

Reply:
Done